viewmodels/src/commonMain/kotlin/com/mirego/trikot/viewmodels/mutable/MutablePickerViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.mirego.trikot.viewmodels.mutable
+
+import com.mirego.trikot.streams.reactive.Publishers
+import com.mirego.trikot.viewmodels.PickerItemViewModel
+import com.mirego.trikot.viewmodels.PickerViewModel
+import com.mirego.trikot.viewmodels.factory.PropertyFactory
+import com.mirego.trikot.viewmodels.properties.ViewModelAction
+import org.reactivestreams.Publisher
+
+open class MutablePickerViewModel<T> : MutableViewModel(), PickerViewModel<T> {
+    override val selectedElementIndex = Publishers.behaviorSubject(0)
+
+    override fun setSelectedElementIndex(index: Int) {
+        selectedElementIndex.value = index
+    }
+
+    override var elements: Publisher<List<PickerItemViewModel<T>>> = PropertyFactory.create(
+        listOf()
+    )
+
+    override var enabled = PropertyFactory.create(true)
+
+    override var action = Publishers.never<ViewModelAction>()
```

Comment:
Adding a comment to explain that we can't add a "onClickListener" to a "Spinner" on Android would be appropriate (and helpful) here :)